client/src/sections/Fridge/Header/Navbar/Navbar.tsx

```diff
@@ -3,19 +3,33 @@ import {useHistory} from 'react-router-dom';
 import {Button, ButtonGroup} from '@shopify/polaris';
 
 import './Navbar.css';
+import {useDispatch} from "react-redux";
+import {logout} from "../../../../store/users/action";
+import {initialState} from "../../../../store/users/reducers";
```

Comment:
Same thing here with the aliasing. Also, you should not be accessing the store directly like this. If you want the initial state, get it from the redux store directly. You did not implement selectors anywhere, please look at my changes to see what I did.